docs/source/bootstrapping.md

```diff
@@ -60,5 +63,12 @@ You must pass in the `toolchain-account` field so a trust-relationship can be se
 
 The `permission-boundary` filed allows you to attach a policy to the role to act as a [permissions boundary](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html)
 
+
+## Qualifiers for Toolchain Role and Target Roles
+We have added suppprt for the use of a qualifier for the toolchain role and the target role(s).  This is to help segregate target deployments when using a single CICD account as the toolchain account across multiple deployment schemes (ex. DEV, INT, PROD).  Most cases do not need this support, but unique deployment scenarios may require it.
```

Comment:
Need to re-write it. Any enterprise which follows multi account deployments might need to adopt similar deployment plan(s)

Reply:
what would you suggest?  I can remove the line 'Most cases do not need this support'

Reply:
We have added support for the use of a qualifier for naming the Toolchain role and Target account's deployment role(s). This is to help segregate and standardize target deployments when using a multi account structure which has a central shared services (CICD account) as the toolchain account performing deployments across relevant environments (ex. DEV, INT, PROD). `qualifier` can be mainly used if you want to restrict the level of access/action a dev/tester/support team can perform on any target given environment